ui/apps/dashboard/src/components/Onboarding/types.ts

```diff
@@ -14,6 +14,17 @@ export const STEPS_ORDER: OnboardingSteps[] = [
   OnboardingSteps.InvokeFn,
 ];
 
+type ArrayLengthRange<
```

Comment:
This is a dense bit of typing....what does it do? I can't totally follow it.

Reply:
I simplified it a bit 😅 It is to return a number between the the minimum defined (in my case it's gonna be 0 and the max number of steps). This way if we add more steps to the onboarding, I don't have to keep adding numbers to this type 0 | 1 | 2 | 3 | 4 ...